backend/packages/Upgrade/src/api/repositories/AnalyticsRepository.ts

```diff
@@ -467,39 +467,13 @@ export class AnalyticsRepository extends Repository<AnalyticsRepository> {
       .addGroupBy('individualEnrollment.experimentId')
       .addGroupBy('individualEnrollment.conditionId')
       .addGroupBy('individualEnrollment.partitionId')
+      .addGroupBy('individualEnrollment.createdAt')
```

Comment:
We don't need to join tables in the query as we are only getting the data from the `individualEnrollment`  table so do we need so many `addGroupBy`?
```
.leftJoin(
        DecisionPoint,
        'decisionPointData',
        'decisionPointData.experimentId = individualEnrollment.experimentId AND decisionPointData.id = individualEnrollment.partitionId'
      )
```
This part can be removed as the partition data is coming from the `experimentsData`.

Reply:
@VivekFitkariwala  This is necessary because we need the **names** of the site and target for the 'Site' and 'Target' columns. Those are not in the data defined in the the type `ExperimentDetailsForCSVData`. We only get the decisionPointId. To get the names of the site and target we have to join on the DecisionPoint table.